_topic_maps/_topic_map_rosa.yml

```diff
@@ -237,6 +237,220 @@ Topics:
+    File: file-integrity-operator-release-notes
+  - Name: Installing the File Integrity Operator
+    File: file-integrity-operator-installation
+  - Name: Updating the File Integrity Operator
+    File: file-integrity-operator-updating
+  - Name: Understanding the File Integrity Operator
+    File: file-integrity-operator-understanding
+  - Name: Configuring the File Integrity Operator
+    File: file-integrity-operator-configuring
+  - Name: Performing advanced File Integrity Operator tasks
+    File: file-integrity-operator-advanced-usage
+  - Name: Troubleshooting the File Integrity Operator
+    File: file-integrity-operator-troubleshooting
+- Name: Security Profiles Operator
+  Dir: security_profiles_operator
+  Topics:
+  - Name: Security Profiles Operator overview
+    File: spo-overview
+  - Name: Security Profiles Operator release notes
+    File: spo-release-notes
+  - Name: Understanding the Security Profiles Operator
+    File: spo-understanding
+  - Name: Enabling the Security Profiles Operator
+    File: spo-enabling
+  - Name: Managing seccomp profiles
+    File: spo-seccomp
+  - Name: Managing SELinux profiles
+    File: spo-selinux
+  - Name: Advanced Security Profiles Operator tasks
+    File: spo-advanced
+  - Name: Troubleshooting the Security Profiles Operator
+    File: spo-troubleshooting
+  - Name: Uninstalling the Security Profiles Operator
+    File: spo-uninstalling
+- Name: cert-manager Operator for Red Hat OpenShift
+  Dir: cert_manager_operator
+  Distros: openshift-rosa
+  Topics:
+  - Name: cert-manager Operator for Red Hat OpenShift overview
+    File: index
+  - Name: cert-manager Operator for Red Hat OpenShift release notes
+    File: cert-manager-operator-release-notes
+  - Name: Installing the cert-manager Operator for Red Hat OpenShift
+    File: cert-manager-operator-install
+  - Name: Managing certificates with an ACME issuer
+    File: cert-manager-operator-issuer-acme
+  - Name: Enabling monitoring for the cert-manager Operator for Red Hat OpenShift
+    File: cert-manager-monitoring
+  - Name: Configuring the egress proxy for the cert-manager Operator for Red Hat OpenShift
+    File: cert-manager-operator-proxy
+  - Name: Customizing cert-manager by using the cert-manager Operator API fields
+    File: cert-manager-customizing-api-fields
+  - Name: Authenticating the cert-manager Operator with AWS Security Token Service
+    File: cert-manager-authenticate-aws
+  - Name: Configuring log levels for cert-manager and the cert-manager Operator for Red Hat OpenShift
+    File: cert-manager-log-levels
+  - Name: Authenticating the cert-manager Operator for Red Hat OpenShift with GCP Workload Identity
+    File: cert-manager-authenticate-gcp
+  - Name: Authenticating the cert-manager Operator for Red Hat OpenShift on AWS
+    File: cert-manager-authentication-non-sts
+  - Name: Authenticating the cert-manager Operator for Red Hat OpenShift on GCP
+    File: cert-manager-authenticate-non-sts-gcp
```

Comment:
It is ROSA document for ROSA cluster, then GCP documents `cert-manager-authenticate-gcp` and `cert-manager-authenticate-non-sts-gcp` are not applicable, need be removed.
`cert-manager-operator-issuer-acme` has GCP ambient credential section, that is also not applicable for ROSA cluster.